src/helm/benchmark/static/schema.yaml

```diff
@@ -1909,7 +1911,7 @@ run_groups:
     subgroups:
       - synthetic_efficiency
     adapter_keys_shown:
-      - model
+      - model_deployment
```

Comment:
I think these should all be `model` as per last discussion.

Reply:
Actually I think we need both

Reply:
Okay, let's try this with both and see if it looks good.